test/lib/oauth.js

```diff
@@ -382,7 +397,8 @@ var setupAppConfig = function(config, callback) {
 
     var dummy = {
         close: function() {
-            child.kill();
+            child.kill("SIGKILL"); // SIGKILL really needed??
+            child.disconnect();
```

Comment:
IIRC the way that Pump.io forks off child workers causes it to have a communication channel with the parent (i.e. it uses [`process.fork()`][1]). I think the fact that you're sending `SIGKILL` is preventing the child from shutting down that channel with the parent, so it stays in a zombie state until you `.disconnect()`.

Hopefully using `.kill()` instead of `.kill('SIGTERM')` will help with this.

 [1]: https://nodejs.org/docs/v0.10.25/api/child_process.html#child_process_child_process_fork_modulepath_args_options